### DISCUSSION TITLE
How to contribute to this project as there is no Issue Tab.

### DISCUSSION THREAD

**Discussion by MohitGupta121** [metabrainz/musicbrainz-android#134] [Q&A]
Please anyone tell me:-

- How to pick new issue as there is no Issue tab showing to me.

**Comment by 07jasjeet** [comment]
Hello, issues are found here: https://tickets.metabrainz.org/secure/RapidBoard.jspa?rapidView=10&projectKey=MOBILE
You can connect with the rest of the team on #metabrainz channel on Libera.chat IRC server.
